recipe/meta.yaml

```diff
@@ -12,47 +12,50 @@ source:
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 2
+  number: 3
 
 requirements:
   host:
     - python =3.9
     - pip
   run:
+    - bottleneck =1.3
+    - cartopy =0.20
+    - cfgrib =0.9.9
```

Comment:
@emanuel-schmid This is the only instance where the bugfix version is also fixed. Is this intentional?

Reply:
@peanutfun This is the only package where semver is not used or not used in the usual way: the package installed is 0.9.9.1